Source/Lib/Codec/EbEncHandle.c

```diff
@@ -401,12 +402,16 @@ EB_ERRORTYPE InitThreadManagmentParams(){
                 char* p = line + physical_id_len;
                 while(*p < '0' || *p > '9') p++;
                 socket_id = strtol(p, NULL, 0);
-                if (socket_id < 0 || socket_id > 15) {
+                if (socket_id < 0) {
                     fclose(fin);
                     return EB_ErrorInsufficientResources;
                 }
                 if (socket_id + 1 > numGroups)
                     numGroups = socket_id + 1;
+                if (socket_id > maxSize) {
+                    maxSize = maxSize*2;
+                    lpGroup = realloc(lpGroup,maxSize*sizeof(processorGroup));
```

Comment:
check lpGroup pointer

Reply:
What do you mean check pointer? Error check? Like this ?
https://github.com/OpenVisualCloud/SVT-HEVC/blob/da7318f6b52df6237e3b33f2285fa9fcd1a51cee/Source/Lib/Codec/EbEncHandle.c#L449-L450

Reply:
Yes, I mean validate the pointer.